api service integration changes (#5958)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/uapiservice/test-instance.md

2. On the browser redirected **Authorize integration** page, click **Install & Authorize**.
3. Copy the client secret from the dialog and store it securely.

>     > **Note**: You must copy the client secret now. For security, it is only displayed once and cannot be retrieved later. Only a masked value will be available after you close this dialog.

Reviewer: ```suggestion
    > **Note**: You must copy the client secret from this dialog. For security purposes, it's only displayed once and can't be retrieved later.
```

4. Click **Done**.
5. On the **General** tab, copy the **Okta domain** and **client ID** and securely store them for your integration test.
6. Configure your app using the Okta domain, client ID, and client secret.

> 7. Perform manual testing to verify your integration.

Reviewer: This is a big ask of the user to verify their integration without any guidance on how to test it.
